### ISSUE TITLE
Data partition of bucket object using chunk_n may not be disjoint.

### ISSUE THREAD

**Issue by gfinol** [lithops-cloud/lithops#447]
When using the `FunctionExecutor.map` function over an obj from a bucket, [`map_function`](https://github.com/lithops-cloud/lithops/blob/master/docs/api-details.md#executormap) receives repeated data when using the parameter `chunk_n` instead of `chunk_size`. 

Data chunks are not disjoint, but they should.

Code to reproduce the bug:

```python
import lithops
import itertools

# Just reading the stream and return lines as a list
def my_map_function(obj):
    lines = obj.data_stream.read().splitlines()
    return lines

# Some file with a bunch of lines (bug seen with 835 lines) 
file =  'bucket_name/extension/file_name.txt'

# When chunk_n is modified, the number of repetitions in the map also changes.
chunk_n = 40

fexec = lithops.FunctionExecutor()
fexec.map(my_map_function, file, chunk_n=chunk_n, runtime_memory=256, timeout=600)
res = fexec.get_result()

# Joining the result of every invocation.
wrap = list( itertools.chain.from_iterable(res) )

# Printing repetitions
for i,line in enumerate(wrap):
	if(wrap.count(line) > 1):
		print(f'Repetition found at position {i}, with content "{line}"')

print( f'{len(list(wrap))} total lines mapped from the file')
```

**Comment by JosepSampe** [comment]
@gfinol Can you upload a link to a specific data file?

**Comment by gfinol** [comment]
@JosepSampe  [Here](https://drive.google.com/file/d/1xqor0QMTIkzPcMcTtS2T8fF50KFNrgTS/view?usp=sharing) you have an example.

**Comment by JosepSampe** [comment]
Fixed in #455.
